src/vsphere_cpi/spec/unit/cloud/vsphere/network_spec.rb

```diff
@@ -319,22 +327,25 @@ module VSphereCloud
                .with('switch-id').and_return(logical_switch)
             expect{
               network.destroy('switch-id')
-            }.to raise_error('Expected switch switch-id to have only one port. Got 0')
+            }.to raise_error
```

Comment:
using raise error without specifying a specific error or error message could make this test pass even when something else raises an error. So, it is best to put the error message back here